lib/gcloud/bigquery/table.rb

```diff
@@ -700,29 +708,20 @@ def extract extract_url, options = {}
       #
       # === A note about large direct uploads
       #
-      # You may encounter a Broken pipe (Errno::EPIPE) error when attempting to
```

Comment:
I'm pretty sure @gramos74 will recognize these doc lines that I added yesterday, since he helped us find the issue!